src/ArrayUtils.php

```diff
@@ -9,17 +9,18 @@
 
 namespace Zend\Stdlib;
 
+use ScriptFUSION\StaticClass;
 use Traversable;
 use Zend\Stdlib\ArrayUtils\MergeRemoveKey;
 use Zend\Stdlib\ArrayUtils\MergeReplaceKeyInterface;
 
 /**
  * Utility class for testing and manipulation of PHP arrays.
- *
- * Declared abstract, as we have no need for instantiation.
  */
-abstract class ArrayUtils
+final class ArrayUtils
 {
+    use StaticClass;
```

Comment:
Yes, and it's an additional dependency and autoloaded trait (not to mention all the mess that traits lead to at reflection level) on a very, very core dependency.

A private constructor is more expressive than an imported trait in this case.